rest-api-spec/src/main/resources/rest-api-spec/api/ml.put_trained_model_definition_part.json

```diff
@@ -0,0 +1,38 @@
+{
+  "ml.put_trained_model_definition_part":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/put-trained-model-definition-part.html",
+      "description":"Adds part of a trained model definition"
+    },
+    "stability":"experimental",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_ml/trained_models/{model_id}/definition/{part_num}",
+          "methods":[
+            "PUT"
+          ],
+          "parts":{
+            "model_id":{
+              "type":"string",
+              "description":"The ID of the trained model for this definition part"
+            },
+            "part_num":{
```

Comment:
Thoughts on this being `part_number`? Also this should be `type: integer` or `type: long` instead of `number`.

Reply:
Actually, the parameter is just called `part`, which seems better to me. Having it `part_num` here is not correct. I will make it an `integer`